clipboard/android/src/main/java/com/capacitorjs/plugins/clipboard/ClipboardPlugin.java

```diff
@@ -0,0 +1,52 @@
+package com.capacitorjs.plugins.clipboard;
+
+import com.getcapacitor.JSObject;
+import com.getcapacitor.NativePlugin;
+import com.getcapacitor.Plugin;
+import com.getcapacitor.PluginCall;
+import com.getcapacitor.PluginMethod;
+
+@NativePlugin(name = "Clipboard")
+public class ClipboardPlugin extends Plugin {
+    private Clipboard implementation;
+
+    @Override
+    public void load() {
+        implementation = new Clipboard(getContext());
+    }
+
+    @PluginMethod
+    public void write(PluginCall call) {
+        String strVal = call.getString("string");
+        String imageVal = call.getString("image");
+        String urlVal = call.getString("url");
+        String label = call.getString("label");
+
+        if (strVal != null) {
+            implementation.write(label, Clipboard.ContentType.STRING, strVal);
+        } else if (imageVal != null) {
+            implementation.write(label, Clipboard.ContentType.IMAGE, imageVal);
+        } else if (urlVal != null) {
+            implementation.write(label, Clipboard.ContentType.URL, urlVal);
+        }
```

Comment:
```suggestion
        if (strVal != null) {
            implementation.write(label, strVal);
        } else if (imageVal != null) {
            implementation.write(label, imageVal);
        } else if (urlVal != null) {
            implementation.write(label, urlVal);
        }
```
The content type is not needed, we always write as text